package.json

```diff
@@ -5,9 +5,11 @@
   "main": "./dist/store.js",
   "scripts": {
     "test": "npm run build_test && jasmine",
+    "clean-ng2": "rimraf node_modules/angular2/manual_typings node_modules/angular2/typings",
     "clean_test": "rm -rf tmp/",
     "build_test": "npm run clean_test && tsc -p spec",
     "build_dist": "rm -rf dist && tsc",
+    "postinstall": "typings install && npm run clean-ng2",
```

Comment:
this seems like a bad idea, as it introduces a dependency on `typings` for consumers of the module, no?